### ISSUE TITLE
Sync Failed - No rights on Outlook calendar folder

### ISSUE THREAD

**Issue by zeppelingen** [phw198/OutlookGoogleCalendarSync#289]
Hi, I've just set up the  GoogleCalendarSync and the sync process generates the following error: "Unable to reclaim orphan calendar entries in Outlook calendar"

**Comment by phw198** [comment]
Have you upgraded to the latest alpha release of OGCS?

:spiral_notepad: Please provide your OGcalsync.log file [[see instructions](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#posting-your-ogcalsynclog-file)]

**Comment by zeppelingen** [comment]
Thanks. 
Yes, I've done it.
Please, find below my log file.
[OGcalsync.txt](https://github.com/phw198/OutlookGoogleCalendarSync/files/1211034/OGcalsync.txt)

**Comment by phw198** [comment]
Error extract:-
```
2017-08-09 11:11:34,223 INFO  6 OutlookGoogleCalendarSync.MainForm [0] -  WARNING: New event failed to save.
Lo sentimos, algo no ha salido bien. Inténtelo de nuevo.
2017-08-09 11:11:34,223 ERROR  6 OutlookGoogleCalendarSync.GoogleCalendar [0] -     en Microsoft.Office.Interop.Outlook.UserProperties.Add(String Name, OlUserPropertyType Type, Object AddToFolderFields, Object DisplayFormat)
   en OutlookGoogleCalendarSync.OutlookCalendar.addOGCSproperty(AppointmentItem& ai, MetadataId key, String value)
   en OutlookGoogleCalendarSync.OutlookCalendar.AddGoogleIDs(AppointmentItem& ai, Event ev)
   en OutlookGoogleCalendarSync.GoogleCalendar.createCalendarEntry_save(Event ev, AppointmentItem ai)
   en OutlookGoogleCalendarSync.GoogleCalendar.CreateCalendarEntries(List`1 appointments)
```

:package: Please could you see if this resolved with hotfix [v2.5.3.3.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/1212986/v2.5.3.3.zip) [[see instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix)].

**Comment by zeppelingen** [comment]
It worked!
Thanks
